### PR TITLE
add rantjs w/ git autoupdate

### LINKED ISSUE #7698: [Request] rantjs
**Library name:RantJS** 
**Git repository url:https://github.com/svenanders/Rantjs**
**npm package url(optional):https://www.npmjs.com/package/rantjs** 
**License(s):unknown**
**Official homepage:https://github.com/svenanders/rantjs**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7698 
@x09326 Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/svenanders/Rantjs
 * Official website (optional, not the repository): http://www.robbestad.com/blog/procedurally-generated-text-with-rantjs
 * NPM package url (optional): https://www.npmjs.com/package/rantjs
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 179
   - Count of watchers: 10
   - Count of forks: 8
 * NPM download stats (optional):
   - Downloads in the last day: 719
   - Downloads in the last week: 734
   - Downloads in the last month: 1003

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7698, cc @svenanders

### CODE REVIEW

**Comment by x09326** [comment]
The first line of the commit is more than 50 chars.

**Comment by pvnr0082t** [comment]
@x09326 I have updated commit message. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
Is the `index*.js` the same as `rant*.js` under https://github.com/svenanders/Rantjs/tree/master/dist ? I prefer to use dist folder and the name with meaning.

**Comment by pvnr0082t** [comment]
@PeterDaveHello `index*.js` and `rant*.js` are totally the same. so how could I do? Grab files from Github? But there is no any tagged release on GitHub. Should I open an issue to ask for adding git tag?

**Comment by PeterDaveHello** [comment]
@pvnr0082t Yeah we can discuss with the author.

**Comment by pvnr0082t** [comment]
@x09326 I have grabbed files from GitHub and replace npm auto-update with git auto-update. The latest version is v1.0.6. Please help me review again, thank you.

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

File: ajax/libs/rantjs/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "rant",
+    "procedural",
+    "text",
+    "generator",
+    "javascript",
+    "random",
+    "text",
+    "sentences"
+  ],
+  "homepage": "http://www.robbestad.com/blog/procedurally-generated-text-with-rantjs",
```
Comment (x09326) [inline ajax/libs/rantjs/package.json]: Please use https instead of http

**Comment by pvnr0082t** [comment]
@x09326 I have updated homepage field. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t since there are much more versions on npm, would you please manually grab the old versions from npm with filename change? Thanks.

**Comment by pvnr0082t** [comment]
@x09326 I have add old versions from npm with filename change. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t please do not generate the map file since v1.0.4/1.0.5 were officially minified, we try to keep the consistency, the map file could be added in new versions only.

**Comment by pvnr0082t** [comment]
@x09326 I have minified without map file. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
Looks great, one last thing @pvnr0082t, to keep the consistency, let's add symbolic links for those versions do not have `rant.min.js`, so that the users can always use the name `rant.min.js` between different versions.

**Comment by pvnr0082t** [comment]
@PeterDaveHello copy `rant.js` to a new file named `rant.min.js` and link `rant.js` to `rant.min.js`?

**Comment by PeterDaveHello** [comment]
@pvnr0082t or just add a symbolic link rant.min.js -> rant.js, thanks.

**Comment by pvnr0082t** [comment]
@x09326 I have added symbolic link for those versions do not have `rant.min.js`. Please help me review again, thank you.

**Pull request merged**